cli/script/command-parser.ts

```diff
@@ -4,7 +4,7 @@ import * as semver from "semver";
 import * as chalk from "chalk";
 
 const USAGE_PREFIX = "Usage: code-push";
-const CODE_PUSH_URL = "https://codepush.azurewebsites.net";
+const CODE_PUSH_URL = "http://codepush.azurewebsites.net";
```

Comment:
My mistake. I must have changed it when testing against localhost, and forgot to check that I changed it back to the right value.